packages.yml

```diff
@@ -3,7 +3,7 @@ packages:
     version: [">=0.2.0", "<0.3.0"]
 
   - package: fivetran/snapchat_ads
-    version: [">=0.5.0", "<0.6.0"]
+    version: [">=0.5.1", "<0.6.0"]
```

Comment:
Hi @fivetran-joemarkiewicz, thanks for the update. The reason I changed the minor version here was that my concern was that this update + the current 0.5.0 version of the snapchat package would break because then certain columns would not be available and cannot be selected with this field mapping: 
```field_mapping={
                'account_id': 'ad_account_id',
                'account_name': 'ad_account_name',
                'ad_group_id': 'ad_squad_id',
                'ad_group_name': 'ad_squad_name',
                'clicks':'swipes'
            },```